MapboxNavigation/Resources/Base.lproj/Navigation.storyboard

```diff
@@ -106,7 +106,6 @@
                                     <constraint firstItem="drb-do-FoT" firstAttribute="centerX" secondItem="nNr-30-cGD" secondAttribute="centerX" id="TCg-jZ-7G3"/>
                                 </constraints>
                                 <userDefinedRuntimeAttributes>
-                                    <userDefinedRuntimeAttribute type="boolean" keyPath="showsUserLocation" value="YES"/>
```

Comment:
Setting this to true prevents the developer from setting map location settings in viewDidLoad.